### PR TITLE
first

### PR DESCRIPTION
**Pull request by Valentin-1988**
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988npm test.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
plese, fix your DEMO link, so that we can start the review

**Comment by Valentin-1988** [comment]
[DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
[TEST REPORT LINK](https:/Valentin-1988.github.io/layout_moyo-header/report/html_report/)

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
your test report link still doesn't works

**Comment by Valentin-1988** [comment]
Як це можливо виправити?

чт, 8 груд. 2022, 09:55 користувач Tatiana Ivanchenko <
***@***.***> пише:

**Comment by Valentin-1988** [comment]
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -1 +1,83 @@
...
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav_list {
+  display: flex;
+}
+
+.nav_item {
```
Comment (TarasHoliuk) [inline src/style.css:35]:
```suggestion
.nav_item:not(:last-child) {
```
and then you can remove it below:
```
.nav_item:last-child {
  margin-right: 0;
}
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,83 @@
...
+  display: block;
+  position: absolute;
+
+  bottom: 0;
+  height: 4px;
+  width: 100%;
+
+  background-color: #00acdc;
+  border-radius: 8px;
+}
```
Comment (TarasHoliuk) [inline src/style.css:83]: add empty line in the end of file

File: src/style.css [hunk 3]
```diff
@@ -1 +1,83 @@
...
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  font-family: Roboto, sans-serif;
```
Comment (TarasHoliuk) [inline src/style.css:45]: Roboto font doesn't exist by default
You should add it

That's a reason tests fail

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ! but please, add Roboto font to your page

**Comment by Valentin-1988** [comment]
Roboto font doesn't exist by default
You should add it як це зробити?

чт, 8 груд. 2022 р. о 20:25 Taras Holiuk ***@***.***> пише:

**Comment by Valentin-1988** [comment]
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988.github.io/layout_moyo-header/report/html_report/)

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
image
image

please, change your header according to design

**Comment by Valentin-1988** [comment]
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988.github.io/layout_moyo-header/report/html_report/)

**Comment by Valentin-1988** [comment]
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988.github.io/layout_moyo-header/report/html_report/)

**Comment by Valentin-1988** [comment]
[DEMO LINK](https://valentin-1988.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://valentin-1988.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 4]
```diff
@@ -1 +1,83 @@
...
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  position: sticky;
```
Comment (AlisaVlasova) [inline src/style.css:21]: you don`t need sticky in this task and it doesn`t work without top prop

File: src/style.css (hunk 1 above)
Comment (AlisaVlasova) [reply src/style.css:35]: ++

File: src/style.css [hunk 5]
```diff
@@ -1 +1,83 @@
...
+  justify-content: center;
+  text-align: center;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  text-transform: uppercase;
+
+  height: 60px;
```
Comment (AlisaVlasova) [inline src/style.css:51]: do not hardcode height, use. vertical paddings

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Comment by Valentin-1988** [comment]
- [DEMO LINK](https://Valentin-1988.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Valentin-1988.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**